### ISSUE TITLE
Malwarepatrol configuration for commercial subscription

### ISSUE THREAD

**Issue by alessiocecchi** [extremeshok/clamav-unofficial-sigs#20]
Hi,

I have a commercial subscription with malwarepatrol and for works with your script I have to modify the URL of signature to:

https://lists.malwarepatrol.net/cgi/getfile?product=15&list=clamav_basic
The difference is the "product" id from 8 to 15.
Thanks

**Comment by extremeshok** [comment]
Thanks for the information,

 i have added the following to config version 52 the version used with the  just released 4.8 release
# Set to no to enable the commercial subscription url.
malwarepatrol_free="yes"

**Comment by ovizii** [comment]
Sorry to add to this old thread but its the best fit. 

Here is my config:
```
malwarepatrol_receipt_code="xxx"
malwarepatrol_product_code="15"
malwarepatrol_list="clamav_ext"
malwarepatrol_free="no"
```

and by looking at: https://www.malwarepatrol.net/open-source.shtml 
I expected to get updates every hour or am I misreading what the benefits of the Premium version offer? but in the logs:  /var/log/clamav-unofficial-sigs/clamav-unofficial-sigs.log I still see: 
`Jan 25 14:35:34 24 hours have not yet elapsed since the last MalwarePatrol download`
